src/pages/sohee/Main/Comments.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import './Comments.scss';
+
+  function Comments({ comment }) {  
+
+  return (
+    // <div className="feed-commentlist">
+    // <div className="feed-comments" key={comment.id}>
+    // <span className="comment-name">{comment.name}</span>
+    // <span >{comment.content}</span>
+    // </div>
+    // </div>
+    <div className="new_comment">
+      <p>
+        {<span className="bold">Smiles</span>}
```

Comment:
```suggestion
        <span className="bold">Smiles</span>
```
변수가 아니어서, { }를 지워도 괜찮을 것 같아요!

Reply:
넵! 알려주신대로 수정하겠습니다. 감사합니다 지현님~!